src/endpoints/mod_feedback.rs

```diff
@@ -112,34 +109,31 @@ pub async fn post_mod_feedback(
         .await
         .or(Err(ApiError::TransactionError))?;
 
-    Ok(HttpResponse::NoContent())
+    Ok(web::Json(ApiResponse {
+        error: "".to_string(),
+        payload: result?,
+    }))
 }
 
 #[delete("/v1/mods/{id}/versions/{version}/feedback")]
 pub async fn delete_mod_feedback(
     data: web::Data<AppData>,
     path: web::Path<GetModFeedbackPath>,
+    payload: web::Json<DeleteModFeedbackPayload>,
```

Comment:
instead of payload, put the id as part of the path:
`/v1/mods/{id}/versions/{version}/feedback/{feedback_id}`

this is more consistent with the rest of the api and also consistent with what a delete request is

Reply:
yeah ok